melos.yaml

```diff
@@ -38,18 +38,6 @@ scripts:
   analyze: >
     dart analyze --fatal-infos . &&
     dart run custom_lint --fatal-infos .
-  test: >
-    melos run test:dart &&
-    melos run test:flutter
-  test:dart: >
```

Comment:
Please leave the separate flutter and dart test commands.

I find them very useful as most of the time I work on neon and and don't need to run the time consuming nextcloud tests locally